### PR TITLE
Show auth error message on Login screen

### PR DESCRIPTION
**Pull request by feedmeapples**
## What was changed

Shows error message on Login

image

## Why?

help users figure out what went wrong

## Checklist

1. Closes 

2. How was this tested:

3. Any docs updates needed?

### CODE REVIEW

File: src/lib/holocene/button.svelte [hunk 1]
```diff
@@ -44,7 +44,7 @@
     <slot />
     {#if count > 0}
       <Badge
-        class="badge absolute top-0 right-0 origin-bottom-left origin-bottom-left translate-y-[-10px] translate-x-[10px]"
+        class="badge absolute top-0 right-0 origin-bottom-left translate-y-[-10px] translate-x-[10px]"
```
Comment (stevekinney) [inline src/lib/holocene/button.svelte:47]: I don't love that we're hardcoding values here, but you didn't do that. My. bigger question is do we know _why_ we were using `origin-bottom-left` and are we sure it's not going to break something else?

File: src/lib/utilities/handle-error.ts [hunk 2]
```diff
@@ -12,12 +13,12 @@ export const handleError = (
   isBrowser = browser,
 ): void => {
   if (isUnauthorized(error) && isBrowser) {
-    window.location.assign(routeForLoginPage());
+    window.location.assign(routeForLoginPage(error?.['message']));
```
Comment (stevekinney) [inline src/lib/utilities/handle-error.ts:16]: Why not `error?.message`?

**Review by stevekinney — APPROVED** [review]

File: src/lib/holocene/button.svelte (hunk 1 above)
Comment (Alex-Tideman) [reply src/lib/holocene/button.svelte:47]: `origin-bottom-left` was a duplicate class in this case, I think the formatter caught it. It's the positioning of the count badge if you pass a count prop to a button.

File: src/lib/utilities/handle-error.ts (hunk 2 above)
Comment (feedmeapples) [reply src/lib/utilities/handle-error.ts]: changed the error type back to `any` so it allows to use `error?.message`

**Pull request merged**